pysat/utils/files.py

```diff
@@ -240,47 +284,74 @@ def parse_delimited_filenames(files, format_str, delimiter):
         return stored
 
     # Parse format string to get information needed to parse filenames
-    search_dict = construct_searchstring_from_format(format_str, wildcard=True)
+    search_dict = construct_searchstring_from_format(format_str, wildcard=False)
     snips = search_dict['string_blocks']
     keys = search_dict['keys']
+    lengths = search_dict['lengths']
```

Comment:
Stylistically, is there a reason these are used instead of just using `search_dict`?  The extra variables were harder for me to track and none of the lines seem particularly long.

Reply:
Updated